src/components/NavBar/ChainSelectorRow.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { ChainId } from '@uniswap/sdk-core'
+import { render } from 'test-utils/render'
+
+import ChainSelectorRow from './ChainSelectorRow'
+
+describe('ChainSelectorRow', () => {
+  it('should match snapshot', () => {
+    const { container } = render(
+      <ChainSelectorRow targetChain={ChainId.OPTIMISM} onSelectChain={jest.fn()} isPending={false} disabled={false} />
```

Comment:
nit: would be good to iterate through all possible targetChains for snapshot matching instead of only Optimism